### PR TITLE
grafana: add traffic metrics for staleread traffic

### LINKED ISSUE #43325: Add stale-read traffic metrics
## Enhancement

Stale-read is used to reducing the cross-region or cross-AZ traffic, thus the hit-rate and traffic caused by stale-read miss is important for us, add the ralated metrics of it.

### PR DESCRIPTION
**Pull request by you06**
### What problem does this PR solve?

Issue Number: close #43325

Problem Summary:

### What is changed and how it works?

With tikv/client-go#776, we can display the following metrics:

Before 15:15, the stale-read hit ratio is high, and after 15:15, there are over 80% stale-read misses. From the metrics, we can clearly find the traffic used by stale-read.

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
Add traffic metrics for stale-read.
```

### CODE REVIEW

File: metrics/grafana/tidb.json [hunk 1]
```diff
@@ -10200,6 +10200,217 @@
...
+              "step": 30
+            },
+            {
+              "exemplar": true,
+              "expr": "sum(rate(tidb_tikvclient_stale_read_bytes_sum{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", direction=\"in\"}[1m])) by (result)",
+              "format": "time_series",
+              "hide": false,
+              "interval": "",
+              "intervalFactor": 2,
+              "legendFormat": "{{result}}-in",
```
Comment (zyguan) [inline metrics/grafana/tidb.json:10367]: How about set expr as `sum(...) by (result, direction)` and use `{{result}}-{{direction}}` here?

**Review by zyguan — APPROVED** [review]

**Review by cfzjywxk — APPROVED** [review]

**Pull request merged**